tests/optimize/test_spsa.py

```diff
@@ -442,6 +442,7 @@ def cost(params):
         ):
             opt.step(cost, params)
 
+    @pytest.mark.xfail(reason="Lightning cannot use adjoint with state prep ops yet")
```

Comment:
TODO: Remove after [#661](https://github.com/PennyLaneAI/pennylane-lightning/pull/661) is merged.